src/client/renderer/r_model.c

```diff
@@ -77,8 +77,7 @@ r_model_t *R_LoadModel(const char *name) {
 
 		mod->type = format->type;
 
-		mod->mins = Vec3_Mins();
-		mod->maxs = Vec3_Maxs();
+		mod->bounds = Bounds_Infinity();
```

Comment:
This could probably be just Box() imo. It's not really infinity, right? It's +/-MAX_WORLD_COORD in all directions?

Reply:
I'm down to rename this to Box(). It *is* infinity, but perhaps Box() makes sense for a "this box literally contains nothing, not even 0,0,0, it's space that cannot physically exist in reality".